retry/src/main/java/io/micronaut/retry/intercept/CircuitBreakerRetry.java

```diff
@@ -188,7 +188,7 @@ private CircuitState openCircuit(Throwable cause) {
             throw new IllegalArgumentException("Exception cause cannot be null");
         }
         if (LOG.isDebugEnabled()) {
-            LOG.debug("Opening Circuit Breaker [{}] due to error: {}", method, cause.getMessage());
+            LOG.debug("Opening Circuit Breaker [{}] due to error: {}", method, cause.getMessage(), cause);
```

Comment:
In retry stacktraces add noise.
```suggestion
            LOG.debug("Opening Circuit Breaker [{}] due to error: {}", method, cause.getMessage());
```